storage/client_merge_test.go

```diff
@@ -198,7 +198,8 @@ func TestStoreRangeMergeLastRange(t *testing.T) {
 
 // TestStoreRangeMergeNonConsecutive attempts to merge two ranges
 // that are not on same store.
-func TestStoreRangeMergeNonConsecutive(t *testing.T) {
+// TODO(tschottdorf): debug the test failures on CircleCI, see #893.
+func disabledTestStoreRangeMergeNonConsecutive(t *testing.T) {
```

Comment:
Can you use `t.Skip("TODO(tschottdorf): debug the test failures on CircleCI, see #893.")` here?